### ISSUE TITLE
Formular editor too wide

### ISSUE THREAD

**Issue by Af0x** [osTicket/osTicket#4696]
Hello,

when you enter user defined formular administration, the list of fields is too wide compared to the rest of the page.

Regards,

Af0x

**Comment by JediKev** [comment]
@Af0x

I don't know what “formular” is so could you please provide screenshots of what you mean?

Cheers.

**Comment by Af0x** [comment]
ehm yes, it´s "Forms" and not Formular.

Admin panel -> Manage -> Forms -> Select a Form

then there is a list of form-fields that is wider than the rest.

**Comment by ntozier** [comment]
I think that they mean

![2019-02-07_1336](https://user-images.githubusercontent.com/1315768/52434501-76284380-2add-11e9-8528-4dc5fafd2bdf.png)

**Comment by Af0x** [comment]
correct. on my screen it´s a bit more offset and even the internal comment field is affected

**Comment by JediKev** [comment]
@Af0x @ntozier 

I do not have this problem. I am using Mac OS and Chrome browser. What does the entire form look like? Is there a Label that's long? Have you cleared cache and/or try a different browser?

Cheers.

**Comment by Af0x** [comment]
I used Firefox 65.0

**Comment by JediKev** [comment]
@Af0x 

Ok...can you post a screenshot of the entire form? Also, have you tried clearing your cache and/or a different browser?

Cheers.

**Comment by ntozier** [comment]
Chrome 72.0.3626.96 (64bit) 

![2019-02-07_1450](https://user-images.githubusercontent.com/1315768/52438696-dfad4f80-2ae7-11e9-9f5a-124ebc7cf26f.png)
![2019-02-07_1451](https://user-images.githubusercontent.com/1315768/52438720-eb991180-2ae7-11e9-90ec-99dfea9f0f79.png)

It's looked this way for a while

**Comment by JediKev** [comment]
@ntozier 

Mine is fine on Chrome but not firefox...let me compare.

Cheers.

**Comment by Af0x** [comment]
that config button uses a lot of space. if you remove the text and stick with the icon-only, maybe it would fit-

**Comment by JediKev** [comment]
@Af0x 

lol that's not the issue, and besides, people would freak out if we removed that text. The issue is the _total_ width; in Chrome there is built in CSS for tables (at least for macs) and it looks perfect but in firefox there is no built in CSS for tables so it runs off a little. The only thing we need to do is add a set width. I'll update this shortly when the pull has been made.

Cheers.

**Comment by Af0x** [comment]
it was just an idea :D

**Comment by ntozier** [comment]
It looks fine on Edge (42.17134.1.0).
Vivaldi (1.10.x) and Vivaldi (2.3.x) looks like my screen shots.
Firefox (64.0) and Firefox (65.0) looks like my screen shots
